test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestfromCsproj.cs

```diff
@@ -590,30 +590,6 @@ public void ItShouldShowWarningMessageOnCollectCodeCoverage()
         {
             var testProjectDirectory = CopyAndRestoreVSTestDotNetCoreTestApp("13");
 
-            // Call test
-            CommandResult result = new DotnetTestCommand(Log, disableNewOutput: true)
-                                        .WithWorkingDirectory(testProjectDirectory)
-                                        .Execute(
-                                            "--collect", "Code Coverage",
-                                            "--filter", "VSTestPassTest");
-
-            // Verify test results
-            if (!TestContext.IsLocalized())
-            {
-                result.StdOut.Should().Contain("No code coverage data available. Code coverage is currently supported only on Windows and Linux x64.");
-                result.StdOut.Should().Contain("Total:     1");
-                result.StdOut.Should().Contain("Passed:     1");
-                result.StdOut.Should().NotContain("Failed!");
-            }
-
-            result.ExitCode.Should().Be(0);
-        }
-
-        [PlatformSpecificFact(TestPlatforms.Linux, Skip = "https://github.com/dotnet/sdk/issues/22865")]
-        public void ItShouldShowWarningMessageOnCollectCodeCoverageThatProfilerWasNotInitialized()
-        {
-            var testProjectDirectory = CopyAndRestoreVSTestDotNetCoreTestApp("13");
-
```

Comment:
This test was removed from main [here](https://github.com/dotnet/sdk/pull/23039/files#diff-eba1c668806635fca7c7a8afc989f1c2076fe2615b7c47ba1afe8aa03415819eL597-L621). The same test was also removed from 6.0.2xx [here](https://github.com/dotnet/sdk/pull/23409/files#diff-eba1c668806635fca7c7a8afc989f1c2076fe2615b7c47ba1afe8aa03415819eL526-L550). Then, the branch merge from 6.0.2xx into main [added it back](https://github.com/dotnet/sdk/pull/23532/commits/c4b1be2602f684295730d81f45951e7afc5fb09b#diff-eba1c668806635fca7c7a8afc989f1c2076fe2615b7c47ba1afe8aa03415819eL597-R621). This was likely a merge issue and the test was intended to be removed, since the first PR was indicated as the reason https://github.com/dotnet/sdk/issues/22865 could be resolved. So, we're rightfully removing the test from `main`.